app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardVMStep/PlanWizardVMStep.js

```diff
@@ -49,6 +59,28 @@ class PlanWizardVMStep extends React.Component {
     } = this.props;
     const discoveryMode = vm_choice_radio === 'vms_via_discovery';
 
+    const overwriteCsvConfirmModal = (
+      <ConfirmModal
```

Comment:
There is already an instance of `<ConfirmModal />` and associated actions to use on `Overview` screens.  For an example please see

- [actions import](https://github.com/ManageIQ/manageiq-v2v/blob/master/app/javascript/react/screens/App/Overview/screens/PlanWizard/index.js)

- [confirmation dialog invocation](https://github.com/ManageIQ/manageiq-v2v/blob/master/app/javascript/react/screens/App/Overview/screens/PlanWizard/PlanWizard.js#L137-L150)

It would be nice to reuse this modal to maintain consistency and reduce duplicate code